dev/storage_base.h

```diff
@@ -149,9 +149,9 @@ namespace sqlite_orm {
                     sql.c_str(),
                     [](void* data, int argc, char** argv, char** /*columnName*/) -> int {
                         auto& tableNames_ = *(data_t*)data;
-                        for(int i = 0; i < argc; i++) {
+                        for(int i = 0; i < argc; ++i) {
                             if(argv[i]) {
-                                tableNames_.push_back(argv[i]);
+                                tableNames_.emplace_back(argv[i]);
```

Comment:
why? what's the difference?

Reply:
I bet it doesn't change the amount of heap allocations but ok

Reply:
Correct. It potentially eliminates move operations. It's one of those changes that I did while walking through all the code.